src/contracts/core/StakeRootCompendium.sol

```diff
@@ -242,79 +243,35 @@ contract StakeRootCompendium is StakeRootCompendiumStorage {
 
     /// SET FUNCTIONS
 
-    function setChargePerProof(uint96 _chargePerStrategy, uint96 _chargePerOperatorSet) public onlyOwner  {
+    /// @inheritdoc IStakeRootCompendium
+    function setMaxTotalCharge(uint96 _maxTotalCharge) external onlyOwner {
```

Comment:
I understand that in practice the maxTotalCharge in USD value should only go up and reflects offchain compute capacity, and due to eth price fluctuation would the charge decrease in size denominated in eth. Perhaps add a comment that if the maxTotalCharge decreases, one must also modify the chargePerProof (perStrategy and perOperatorSet) so that the totalCharge does not already exceed the new maxTotalCharge to avoid "brick"-ing operatorset removals / additions.

Reply:
ah. ill add something to not allow reducing below the current totalCharge